README.md

```diff
@@ -147,6 +147,17 @@ If correctly set you should see the following output in your console
 OP-SQLITE performance mode enabled! 🚀
 ```
 
+## Thread safety
+Please, note that OP_SQLITE_PERF=1 will disable Thread Safe support of SQLite, and you will be unable to use async methods from this library, otherwise it will crash.
```

Comment:
Please remove this line, based on adding the comment above.

Reply:
@ospfranco. I've done the documentation changes. Do you think that is useful to mention that with SQLite based thread safety, its possible to run parallel read querys? Or its implicity enough?